operator/pkg/tasks/deint/cert.go

```diff
@@ -0,0 +1,65 @@
+package tasks
```

Comment:
Please update `deint/cert.go` to `deinit/cert.go`

Reply:
updated. @lonelyCZ